src/Result/Row.php

```diff
@@ -40,38 +38,15 @@ public function __get(string $name)
 	}
 
 
-	public function offsetExists($offset)
-	{
-		if (!is_int($offset)) {
-			throw new NotSupportedException('Array access is suported only for indexed reading. Use property access.');
-		}
-
-		return $offset >= 0 && $offset < count((array) $this);
-	}
-
-
-	public function offsetGet($offset)
+	/**
+	 * @return mixed
+	 */
+	public function getColumn(int $offset)
```

Comment:
I think the problem is that most people would expect this method to accept `string $name` as parameter. Also technically you are not returning *column* but a *value of a cell*. So a better name might be `getCellValueByColumnOffset()`.